airbyte/cloud/connections.py

```
@@ -89,6 +89,14 @@ def source(self) -> CloudSource:
         )
         return self._cloud_source_object
 
+    @property
+    def source(self) -> CloudSource:
```

Ruff (0.8.0)

Redefinition of unused `source` from line 81

(F811)